src/main/java/hudson/plugins/git/extensions/impl/SubmoduleOption.java

```diff
@@ -130,7 +130,10 @@ public void onCheckoutCompleted(GitSCM scm, Run<?, ?> build, GitClient git, Task
             if (!disableSubmodules && git.hasGitModules()) {
                 // This ensures we don't miss changes to submodule paths and allows
                 // seamless use of bare and non-bare superproject repositories.
-                git.setupSubmoduleUrls(revToBuild.lastBuild.getRevision(), listener);
+                git.setupSubmoduleUrls(
```

Comment:
Currently the method `setupSubmoduleUrls` only has 2 parameters. Therefore also the build for this pull request is current failing.

May I ask if you missed something?

Reply:
@richardwhiuk did you already had the time to look at this?